app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/ParkingView.java

```diff
@@ -0,0 +1,15 @@
+package com.mobile.tandil.javabaseproject.mvp.view;
+
+import android.app.Activity;
+
+import com.mobile.tandil.javabaseproject.databinding.ActivityMainBinding;
+import com.mobile.tandil.javabaseproject.mvp.contract.ParkingContract;
+import com.mobile.tandil.javabaseproject.mvp.view.base.ActivityView;
+
```

Comment:
remove this extra space please, let only one

Reply:
done